static/js/app.js

```diff
@@ -226,7 +234,7 @@ $(function() {
         },
         fetchMarkers : function() {
             if (!this.isReady) return;
-            this.updateUrl();
```

Comment:
It did reduce the app's loading time. I can't tell exactly by how much, but it feels visually better.